packages/liveblocks-react-lexical/src/comments/floating-composer.tsx

```diff
@@ -114,21 +116,33 @@ export const FloatingComposer = forwardRef<
     });
   }, [editor, showComposer]);
 
+  useEffect(() => {
+    return editor.registerUpdateListener(({ editorState: state, tags }) => {
+      // Ignore selection updates related to collaboration
+      if (tags.has("collaboration")) return;
+      state.read(() => setShowActiveSelection(false));
+    });
+  }, [editor]);
+
   if (!showComposer) return null;
 
   return (
-    <FloatingSelectionContainer
-      sideOffset={5}
-      alignOffset={0}
-      collisionPadding={10}
-    >
-      <Composer
-        autoFocus
-        {...composerProps}
-        onKeyDown={handleKeyDown}
-        onComposerSubmit={handleComposerSubmit}
-        ref={forwardedRef}
-      />
-    </FloatingSelectionContainer>
+    <>
+      {showActiveSelection && <ActiveSelection />}
+      <FloatingSelectionContainer
+        sideOffset={5}
+        alignOffset={0}
+        collisionPadding={10}
+      >
+        <Composer
+          autoFocus
+          {...composerProps}
+          onKeyDown={handleKeyDown}
+          onComposerSubmit={handleComposerSubmit}
+          ref={forwardedRef}
+          onFocus={() => setShowActiveSelection(true)}
```

Comment:
Because we still accept an `onFocus` prop in this component, we should still execute the `onFocus` callback passed as prop before/after executing our callback. We do this for `onComposerSubmit` and `onKeyDown`.